go/tools/microbench/microbench.go

```diff
@@ -190,7 +207,10 @@ func findBenchmarks(loaded []*packages.Package) []benchmark {
 			}
 		}
 	}
-	return benchmarks
+	if len(multierr.Errors(err)) > 0 {
+		return nil, err
+	}
+	return benchmarks, nil
```

Comment:
checking err != nil should be reasonable here.

Reply:
Agreed, fixed in 467093b014ef9bfc7f17b0024d07417caf3e0740.